applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -1004,6 +1119,511 @@ static int cmd_location(const struct shell *shell, size_t argc, char **argv)
+	char *preset_argv[3] = {"preset", "48_2_2", "0"};
+	char *adv_name_argv[3] = {"adv_name", "John's phone", "0"};
+	char *name_argv[3] = {"name", "Personal sharing", "0"};
+	char *packing_argv[3] = {"packing", "int", "0"};
+	char *encrypt_argv[4] = {"encrypt", "1", "0", "Auratest"};
+
+	char *context_argv[4] = {"context", "conversational", "0", "0"};
+
+	char *num_bis_argv[4] = {"num_bises", "2", "0", "0"};
+
+	char *location_FL_argv[5] = {"location", "FL", "0", "0", "0"};
+	char *location_FR_argv[5] = {"location", "FR", "0", "0", "1"};
+
+	cmd_preset(shell, 3, preset_argv);
+	cmd_adv_name(shell, 3, adv_name_argv);
+	cmd_name(shell, 3, name_argv);
+	cmd_packing(shell, 3, packing_argv);
+	cmd_encrypt(shell, 4, encrypt_argv);
+
+	cmd_context(shell, 4, context_argv);
+
+	cmd_num_bises(shell, 4, num_bis_argv);
+
+	cmd_location(shell, 5, location_FL_argv);
+	cmd_location(shell, 5, location_FR_argv);
+
+	cmd_show(shell, 0, NULL);
+}
+
+static void personal_multi_language_set(const struct shell *shell)
+{
+	char *num_subs_argv[3] = {"num_subgroups", "2", "0"};
+	char *preset_argv[3] = {"preset", "24_2_2", "0"};
+	char *adv_name_argv[3] = {"adv_name", "Brian's laptop", "0"};
+	char *name_argv[3] = {"name", "Personal multi-lang.", "0"};
+	char *packing_argv[3] = {"packing", "int", "0"};
+	char *encrypt_argv[4] = {"encrypt", "1", "0", "Auratest"};
+
+	char *lang0_argv[4] = {"lang_set", "eng", "0", "0"};
+	char *lang1_argv[4] = {"lang_set", "spa", "0", "1"};
+
+	char *context0_argv[4] = {"context", "media", "0", "0"};
+	char *context1_argv[4] = {"context", "media", "0", "1"};
+
+	char *num_bis0_argv[4] = {"num_bises", "2", "0", "0"};
+	char *num_bis1_argv[4] = {"num_bises", "2", "0", "1"};
+
+	char *location_FL0_argv[5] = {"location", "FL", "0", "0", "0"};
+	char *location_FR0_argv[5] = {"location", "FR", "0", "0", "1"};
+	char *location_FL1_argv[5] = {"location", "FL", "0", "1", "0"};
+	char *location_FR1_argv[5] = {"location", "FR", "0", "1", "1"};
+
+	cmd_num_subgroups(shell, 3, num_subs_argv);
+	cmd_preset(shell, 3, preset_argv);
+	cmd_adv_name(shell, 3, adv_name_argv);
+	cmd_name(shell, 3, name_argv);
+	cmd_packing(shell, 3, packing_argv);
+	cmd_encrypt(shell, 4, encrypt_argv);
+
+	cmd_lang_set(shell, 4, lang0_argv);
+	cmd_lang_set(shell, 4, lang1_argv);
+
+	cmd_context(shell, 4, context0_argv);
+	cmd_context(shell, 4, context1_argv);
+
+	cmd_num_bises(shell, 4, num_bis0_argv);
+	cmd_num_bises(shell, 4, num_bis1_argv);
+
+	cmd_location(shell, 5, location_FL0_argv);
+	cmd_location(shell, 5, location_FR0_argv);
+	cmd_location(shell, 5, location_FL1_argv);
+	cmd_location(shell, 5, location_FR1_argv);
+
+	cmd_show(shell, 0, NULL);
+}
+
+static int cmd_usecase(const struct shell *shell, size_t argc, char **argv)
+{
+	if ((argc >= 2) && (strcmp(argv[1], "print") == 0)) {
+		usecase_print(shell);
+		return 0;
+	}
+
+	if (argc < 2) {
+		shell_error(shell, "Usage: bct usecase <use_case> - Either index or text");
+		return -EINVAL;
+	}
+
+	int use_case_nr = -1;
```

Comment:
I think the code is more readable if this is declared as the enum. Then if an index is given as shell argument that can be casted to enum instead